paasta_tools/kubernetes_tools.py

```diff
@@ -681,6 +681,19 @@ def registration_label(namespace: str) -> str:
     return f"registrations.{PAASTA_ATTRIBUTE_PREFIX}{limited_namespace}"
 
 
+def contains_zone_label(node_selectors: Dict[str, NodeSelectorConfig]):
+    return any(
+        k in node_selectors
+        for k in (
+            "topology.kubernetes.io/zone",
```

Comment:
creating a constant at the top of file would be nice. otherwise shipit

Reply:
done but can't use `paasta_prefixed` in the constant